src/Gamepage/Gamepage.scss

```diff
@@ -1,24 +1,37 @@
 @mixin allDivRules {
-  border: solid .25px black;
-  background-color: whitesmoke;
-  opacity: .8;
+  border: solid 1.75px #282D33;
+  background-color: white;
+  box-shadow: -1px 1px 13px 0px #282D33;
 }
 
 @mixin bottomDivRules {
   padding-top: 5px;
   padding-bottom: 5px;
-  height: 90%;
-  width: 90%;
-  border-radius: 5%;
-  place-self: center;
+  height: 95%;
+  align-self: center;
+  border-radius: 15px;
   overflow-y: scroll;
   @include allDivRules();
-  box-shadow: 7px 0px 20px 1px #888888;
+}
+
+@mixin chatAndAttempts {
```

Comment:
Thanks for changing the names of theses! I know we have a few other places that could get a name change. Maybe we can go through the whole project as a team (after it's turned in) and brainstorm some name techniques.